### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#828] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello can someone show me away to get around the extension page block for Ltmeat Im currently trying to use: chrome-extension://joflmkccibkooplaeoinecjbmdebglab/manifest.json

**Comment by nathblade16** [comment]
what version are you on

**Reply by Vhirttuegaming** [reply]
114.0.5735.340

**Reply by AshtonDavies** [reply]
If it's blocked by policy, you should try "chrome-extension://joflmkccibkooplaeoinecjbmdebglab/_generated_background_page.html" and if that's also blocked just use the Securly block page. They're not gonna block that lol.

**Reply by Vhirttuegaming** [reply]
How do I use the securly block page?

**Reply by AshtonDavies** [reply]
Step 1: Attempt to navigate to the most inappropriate website you know of.
Step 2: There is no step 2 you're on the block page now.
Use that page instead of the background or manifest page it should be done the same way.

**Reply by Vhirttuegaming** [reply]
chrome-extension://fullblockpageurl?/_generated_background_page.html

**Reply by AshtonDavies** [reply]
You can cut out the BASE64 ?category part and just use the bare url.

**Reply by Vhirttuegaming** [reply]
got it thanks. the most inapproprate site Ik of is coolmathgames.

**Reply by AshtonDavies** [reply]
Lol 😂

**Reply by Vhirttuegaming** [reply, marked as answer]
Actually Securly got updated so it changed its code... I didn't need this much instruction...

**Reply by AshtonDavies** [reply]
Oh yeah, fyi there are two Securly installation versions. The webstore and external. And technically also the classroom one.

(https://chrose.netlify.app?detail=Extension%20IDs)

**Reply by Vhirttuegaming** [reply]
What is the difference between the two?

**Reply by AshtonDavies** [reply]
Essentially just depends how the admin wants it installed and configured I guess. I don't really know, because I'm a Lightspeed user but some of us nerds basically study these extensions and ridiculous things.

**Comment by funkymokey** [comment]
probably shouldn't use your actual chrome extension link

**Reply by Vhirttuegaming** [reply]
y?

**Reply by funkymokey** [reply]
security reasons

**Reply by AshtonDavies** [reply]
That doesn't make any sense if you don't elaborate.
